data_structures/linked_list/add_nodes.py

```diff
@@ -0,0 +1,81 @@
+"""Adding two integers of same length using Linked lists
+
+Example add 321 + 248 = 569
+
+(1)->(2)->(3) + (8)->(4)->(2) = (5)->(6)->(9) """
+
+
+# node definition
+class Node:
+    def __init__(self, data):
+        self.data = data
+        self.next = None
+        # self.head=None
+
+
+# linkedlist definition
+class LL:
+    def __init__(self):
+        self.head = None
+
+    # method to print the linked list
+    def printLL(self):
+        temp = self.head
+        while temp:
+            print(temp.data, end=" ")
+            temp = temp.next
+
+    # method to push a new node onto the linked list
+    def push(self, newdata):
```

Comment:
Please add [Python doctests](https://docs.python.org/3/library/doctest.html) for `.push()` and `.add()` as discussed in [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).